x-pack/legacy/plugins/apm/public/components/app/Settings/ListSettings.tsx

```diff
@@ -0,0 +1,196 @@
+      }),
+      sortable: true,
+      render: (value: string) => value
+    },
+    {
+      name: '',
+      actions: [
+        {
+          name: 'Edit',
+          description: 'Edit this config',
+          icon: 'pencil',
+          color: 'primary',
+          type: 'icon',
+          onClick: (config: Config) => {
+            setSelectedConfig(config);
+            setIsFlyoutOpen(true);
+          }
+        }
+      ]
+    }
+  ];
+
+  const RETURN_TO_OVERVIEW_LINK_LABEL = i18n.translate(
+    'xpack.apm.returnToOverviewLinkLabel',
+    {
+      defaultMessage: 'Return to overview'
+    }
+  );
+
+  const hasConfigurations = data.length !== 0;
+
+  return (
+    <>
+      <AddSettingsFlyout
+        isOpen={isFlyoutOpen}
+        selectedConfig={selectedConfig}
+        onClose={() => {
+          setSelectedConfig(null);
+          setIsFlyoutOpen(false);
+        }}
+        onSubmit={() => {
+          setSelectedConfig(null);
+          setIsFlyoutOpen(false);
+          refresh();
+        }}
+      />
+
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem grow={false}>
+          <EuiTitle size="l">
+            <h1>Settings</h1>
+          </EuiTitle>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <APMLink path="/">
+            <EuiButtonEmpty size="s" color="primary" iconType="arrowLeft">
+              {RETURN_TO_OVERVIEW_LINK_LABEL}
+            </EuiButtonEmpty>
+          </APMLink>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+
+      <EuiSpacer size="l" />
+
+      <EuiPanel>
+        <EuiFlexGroup alignItems="center">
+          <EuiFlexItem grow={false}>
+            <EuiTitle>
+              <h2>Configurations</h2>
+            </EuiTitle>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiBetaBadge
+              label="Beta"
+              tooltipContent="This feature is not GA. Please help us by reporting any bugs."
+            />
+          </EuiFlexItem>
+          {hasConfigurations ? (
+            <EuiFlexItem>
+              <EuiFlexGroup alignItems="center" justifyContent="flexEnd">
+                <EuiFlexItem grow={false}>
+                  <EuiButton
+                    color="primary"
+                    fill
+                    iconType="plusInCircle"
+                    onClick={() => setIsFlyoutOpen(true)}
+                  >
+                    Create configuration
+                  </EuiButton>
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiFlexItem>
+          ) : null}
+        </EuiFlexGroup>
+
+        <EuiSpacer size="m" />
+
+        <EuiCallOut
+          title="Agent Central Configuration (BETA)"
+          iconType="iInCircle"
+        >
+          <p>
+            We're excited to bring you a first look at how you can control your
+            services by fine-tuning agent configurations directly from Kibana.
+            Learn more about this new feature.
```

Comment:
@bmorelli25 We're going to need a URL for the documentation page we're intending to set up for APM Agent Configuration.

Reply:
Let's go with: `https://www.elastic.co/guide/en/kibana/current/agent-configuration.html`

Reply:
Added this in (cc @ogupte) 👍